snippets/base/admin/fields.py

```diff
@@ -78,6 +81,22 @@ def validate(self, value):
         return value
 
 
+class JEXLFirefoxRangeField(JEXLRangeField):
+    def __init__(self, **kwargs):
+        choices = ([(None, 'No limit')] +
+                   [(x, x) for x in version_list(product_details.firefox_history_major_releases)])
```

Comment:
snippets regularly updates the data here? It won't have recent releases otherwise.

Reply:
Yes, we do run `update_product_details` in cron. https://github.com/mozmeao/snippets-service/blob/e65443932992180f4ae417715a3c8f0f2b96dda5/scripts/cron.py#L57